src/app/admin/modals/user-notification-settings-modal/notification.component.ts

```diff
@@ -0,0 +1,95 @@
+import { Component, OnInit, Inject } from '@angular/core';
+import {
+  alertService,
+  currentUser,
+  auth,
+  User
+} from 'src/app/ajs-upgraded-providers';
+import { MatDialogRef } from '@angular/material/dialog';
+import { NgForm, FormBuilder, FormControl, FormGroup } from '@angular/forms'
+
+@Component({
+  selector: 'df-notific-setting',
+  templateUrl: './notification.component.html',
+  styleUrls: ['./user-notification-settings-modal.scss'],
+})
+export class notificationComponent implements OnInit {
+  // User: userModel;
+  feedbackControl = new FormControl();
+  portfolioControl = new FormControl();
+  taskControl = new FormControl();
+  notificationSettings: FormGroup;
+  classList: Array<string>;
+  config = {
+    animated: true, 
+    keyboard: true, 
+    backdrop: true,
+    ignoreBackdropClick: false, 
+  };
+  constructor(
+    public dialogRef: MatDialogRef<notificationComponent>,
+    fb: FormBuilder,
```

Comment:
**Suggestion**: can we rename this variable to "formBuilder"?

Reply:
renamed fb as formBuilder